pages/linux/aplay.md

```diff
@@ -0,0 +1,15 @@
+# aplay
+
+> Command-line sound player for ALSA soundcard driver.
+
+- Play file "foobar". Sampling rate, bit depth, and so forth will be automatically determined for supported soundfile formats:
+
+`aplay {{foobar}}`
+
+- Play first 10 seconds of soundfile at 2500hz:
+
+`aplay -d 10 -r 2500 foobar`
```

Comment:
We prefer the long argument format, is easy to understand what the command is doing.
```suggestion
`aplay --duration={{10}} --rate={{2500}} {{path/to/file}}`
```